include/yaramod/types/regexp.h

```diff
@@ -368,13 +454,22 @@ class RegexpRange : public RegexpOperation
 		return v->visit(this);
 	}
 
-	const std::pair<std::optional<std::uint64_t>, std::optional<std::uint64_t>>& getRange() const
+	std::pair<std::optional<std::uint64_t>, std::optional<std::uint64_t>> getRange() const
 	{
-		return _range;
+		std::optional<std::uint64_t> out1;
+		std::optional<std::uint64_t> out2;
+		if (_first)
+			out1 = std::make_optional(_first.value()->getUInt64_t());
```

Comment:
Is this `make_optional` necessary here?

Reply:
No, `std::optional<T>::operator=` can take `T` directly.